### PR TITLE
Workshop 3: higher-order functions

### PR DESCRIPTION
**Pull request by rvanheest**
@janvanmansum here is the preparation for the third workshop. Would you be able to work through it and give feedback on possible mistakes/typos/etc. Thanks!

### CODE REVIEW

**Comment by rvanheest** [comment]
@janvanmansum please merge

File: src/main/scala/workshop3/assignments/Employee.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Employee
+========
+
+1. In [Employee.sc](Employee.sc) an `Employee` can either be a `Developer` or a `Manager`. Implement the functions
```
Comment (jo-pol) [inline src/main/scala/workshop3/assignments/Employee.md:4]: You git-ignored your sc files so this link is broken

**Review by jo-pol — COMMENTED** [review]
please fix the linked but ignored .sc files

File: src/main/scala/workshop3/assignments/Employee.scala [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	// returns the list of employees, but with a raise for the manager's salary
+	def raiseManagerSalaries(increase: Int, employees: List[Employee]): List[Employee] = ???
+
+	// returns the list of employees, but with a raise for the employee's salary
+	def raiseDeveloperSalaries(increase: Int, employees: List[Employee]): List[Employee] = ???
+
+	// returns the list of employees, but with a percentage raise for the manager's salary
+	def raiseManagerSalariesByPercent(percent: Int, employees: List[Employee]): List[Employee] = ???
+
+	// returns the list of employees, but with a percentage raise for the employee's salary
```
Comment (jo-pol) [inline src/main/scala/workshop3/assignments/Employee.scala:33]: I guess you mean: return all employees but raise only manager salaries

Comment (rvanheest) [reply src/main/scala/workshop3/assignments/Employee.scala:33]: Yes, return a list of all employees, but with a (percentage/absolute) raise for the manager/developer; I'll fix this in a new PR. Thanks!

**Pull request merged**